com.civicrm.hrjobroles/CRM/Hrjobroles/Upgrader.php

```diff
@@ -78,6 +78,25 @@ public function upgrade_1006() {
     return TRUE;
   }
 
+  /**
+   * Creates an option group for organisation contact
+   * used for job role funder
+   *
+   * @return bool
+   */
+  public function upgrade_1007() {
+    $result = civicrm_api3('OptionGroup', 'get', [
+      'name' => 'hrjc_funders',
+    ]);
+
+    if ($result['count'] === 0) {
+      $file = 'xml/option_groups/organisation_contact_install.xml';
```

Comment:
Not sure if `organisation_contact` is the best name for this option group file, one might be mistaken that it stores contacts or something and it's a bit generic. What about, `organisation_provider` or a better name. What do you think?
Same goes for other PR.

Reply:
This is a good idea. 🍭